### PR TITLE
feat: No refresh in global palette

### PR DESCRIPTION
**Pull request by amithire1201**
### Description
https://trello.com/c/pyGM8IEp/948-global-color-palette-no-refresh-in-customizer

### Checklist:
- [x] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by premanshup — APPROVED** [review]

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        
+        {
+            "type": "pwa-chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
+}
```
Comment (premanshup) [inline .vscode/launch.json:16]: @amithire1201 Please check if we need this in Theme repo.

File: functions.php [hunk 2]
```diff
@@ -158,3 +158,10 @@
 require_once ASTRA_THEME_DIR . 'inc/core/deprecated/deprecated-filters.php';
 require_once ASTRA_THEME_DIR . 'inc/core/deprecated/deprecated-hooks.php';
 require_once ASTRA_THEME_DIR . 'inc/core/deprecated/deprecated-functions.php';
+
+
+add_action('wp_default_scripts', function ($scripts) {
+    if (!empty($scripts->registered['jquery'])) {
+        $scripts->registered['jquery']->deps = array_diff($scripts->registered['jquery']->deps, ['jquery-migrate']);
+    }
+});
```
Comment (premanshup) [inline functions.php:167]: @amithire1201 I don't think this is required, please confirm.

**Review by premanshup — CHANGES_REQUESTED** [review]

**Comment by premanshup** [comment]
@amithire1201 On changing the Preset, I'm getting these JS console errors, do check them - https://d.pr/i/OkHea5

File: functions.php (hunk 2 above)
Comment (Nikschavan) [reply functions.php:167]: Also - let's not _yet_ write anonymous PHP functions, I am not sure we have declared Astra to be PHP 5.6+ yet.

File: .vscode/launch.json (hunk 1 above)
Comment (amithire1201) [reply .vscode/launch.json]: Removed.

File: functions.php (hunk 2 above)
Comment (amithire1201) [reply functions.php]: Removed. Added by mistake

**Comment by amithire1201** [comment]
> @amithire1201 On changing the Preset, I'm getting these JS console errors, do check them - https://d.pr/i/OkHea5

Did not reproduced at my end. Could you please mention the steps to reproduce?

**Comment by premanshup** [comment]
> > @amithire1201 On changing the Preset, I'm getting these JS console errors, do check them - https://d.pr/i/OkHea5
> 
> Did not reproduced at my end. Could you please mention the steps to reproduce?

I will share the steps on task. Merging this PR for now, because we need to implement Backward Comp for old palette color.

**Review by premanshup — APPROVED** [review]

**Pull request merged**